docs/contributing.rst

```diff
@@ -179,4 +180,35 @@ To run only a single test type::
    $ PYTHONPATH=. python tests/test_file_to_run.py
 
 .. _`bugs`: https://github.com/pytrip/pytrip/issues
-.. _`features`: https://github.com/pytrip/pytrip/issues
\ No newline at end of file
+.. _`features`: https://github.com/pytrip/pytrip/issues
+
+
+Nomenclature
+------------
+
+1. Classes: CamelTyped. Example: ``class CtxCube()``
+2. Methods and functions: lowercase , typically containing a verb and separated by underscore. Example: ``def save_cube()``
+    * avoid ``get_*`` and ``set_*`` functions as this is not pythonic.
+3. Attributes and variables: lowercase and typically consisting of one or more nouns separated by underscore. Example: ``self.target_dose``
+4. Functions, class methods, attributes etc which are not supposed to be accessed by users should be prefixed with underscore i.e. ``_foobar``
+5. Directories, paths and filenames should be named following this scheme:
+
+* **Filenames**
+    * ``funk.dat`` : filename
+    * ``funk`` : basename
+* **Directories**
+    * ``/home/bassler/foobar`` : absolute directory ``abs_dir``
+    * ``./foobar`` : relative directory ``rel_dir``
+    * or just ``dir`` if both may be applicable.
```

Comment:
But the documentation doesn't mentione prefixing, so I'd either say `_dir` or `dir_` in order not to suggest user to use `dir` name.